redisvl/extensions/router/schema.py

```diff
@@ -16,7 +16,7 @@ class Route(BaseModel):
     """List of reference phrases for the route."""
     metadata: Dict[str, str] = Field(default={})
     """Metadata associated with the route."""
-    distance_threshold: Optional[float] = Field(default=None)
+    distance_threshold: float = Field(default=0.5)
```

Comment:
I saw you deleted another validator, so maybe this isn't important, but with Pydantic 2, you can validate that a number is within a range using `Annotated`:

```
from typing_extensions import Annotated

from pydantic.v1 import BaseModel, Field, ValidationError

class Foo(BaseModel):
    bar: Annotated[float, Field(strict=True, gt=0, lte=1)]  # or gt, lt


for bad_number in (-1, 2):
    try:
        Foo(bar=bad_number) 
    except ValidationError:
        print("worked")
    else:
        raise RuntimeError("Didn't work!")

print(Foo(bar=0.5))
```

Reply:
Oh that's nice

Reply:
Yeah I deleted a validator for the now depreciated field since it has no impact. I think we should update to the new style of validators but I think that should be included in the pr for pydantic versioning stuff because it fits more with that theme: https://redislabs.atlassian.net/browse/RAAE-607